src/components/Success/Success.module.scss

```diff
@@ -0,0 +1,105 @@
+.successComponent{
+    background: #000000;

```

Comment:
The idea is this would be a popup that appears on top, so it doesn't need to have black background.